dataverse-webapp/src/main/webapp/resources/iqbs/fileDownloadButtons.xhtml

```diff
@@ -22,7 +22,7 @@
 
 
 <composite:implementation>
-    <c:set var="downloadsDisabled" value="#{cc.attrs.fileMetadata.dataFile.ingestInProgress or cc.attrs.lockedFromDownload}"/>
+    <c:set var="downloadsDisabled" value="#{cc.attrs.lockedFromDownload}"/>
```

Comment:
IMO just remove `downloadsDisabled` variable. We don't need to check for any locks. This way it would be consistent with multiple download button